content/en/account_management/rbac/permissions.md

```diff
@@ -21,27 +21,17 @@ After creating a role, assign or remove permissions to this role directly by [up
 
 ## Overview
 
-### General permissions
-
-General permissions provide the base level of access for your role. [Advanced Permissions](#advanced-permissions) are explicitly defined permissions that augment the base permissions.
-
-{{< permissions group="General" >}}
-
-**Note**: There is no `read-only` permission as it is defined by the lack of the `standard` permission for a role.
-
-### Advanced permissions
-
 By default, existing users are associated with one of the three out-of-the-box roles:
 
 - Datadog Admin
 - Datadog Standard
-- Datadog Read-Only.
+- Datadog Read-Only
 
-All users can read all data types. Admin and Standard users have write permissions on assets.
+All users can read all data types. Admin and Standard users have write permissions on assets. Admin users have additional read and write permissions for sensitive assets relating to user management, org management, billing, and usage.
```

Comment:
I might change the first sentence to "All users with one of these roles can read all data types". That's just clarify that the default Read-Only role allows read access to all resources (I think?), but custom roles can be crafted to disallow reading certain resources.

Reply:
LGTM! Probably in Stephen's queue.